functions/uppercase/java-boot/pom.xml

```diff
@@ -30,13 +30,14 @@
 	<dependencies>
 		<dependency>
 			<groupId>org.springframework.boot</groupId>
-			<artifactId>spring-boot-starter-web</artifactId>
+			<artifactId>spring-boot-starter</artifactId>
 		</dependency>
+		<!--
```

Comment:
maybe remove this instead of commenting it out. Also remove the `dependencyManagement` section

Reply:
👍